docker-compose.yml

```diff
@@ -231,7 +231,7 @@ services:
         aliases:
          - quickstatements.formulasearchengine.com
```

Comment:
this one needs to be changed.

Reply:
I figured out that simple edits can be performed in the editor directly. This is more efficient than writing comments and requesting the change